examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,148 @@
+# usage:
+#   python letter.py input.csv
+#   logs to csv's in the output directory
+
+import csv
+import datetime
+import lob
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(__file__+'../../..'))
+
+###########################################################
+# TODO: Provide the following constants
```

Comment:
Has this todo been taken care of? This comment probably shouldn't be consumer facing, correct?